releasenotes/notes/composite-exp-transpile-e37c257ba007ff40.yaml

```diff
@@ -0,0 +1,13 @@
+---
+upgrade:
```

Comment:
Looks like you'd want to remove the release notes of #667, since it's covered here